src/map/script.c

```diff
@@ -2862,13 +2862,18 @@ struct script_data *get_val(struct script_state* st, struct script_data* data) {
 
 		switch (prefix) {
 			case '@':
```

Comment:
the cases are on the wrong indentation, they should be on same one was the switch